name-service/program/src/instruction.rs

```diff
@@ -89,6 +89,20 @@ pub enum NameRegistryInstruction {
     ///   2. `[writeable]` Refund account
     ///
     Delete,
+
+    /// Realloc the data of a name record.
+    ///
+    /// The space change cannot be more than `MAX_PERMITTED_DATA_LENGTH` different than
```

Comment:
nit for clarity, since you can reduce by any amount
```suggestion
    /// The space change cannot be more than `MAX_PERMITTED_DATA_LENGTH` greater than
```

Reply:
Ah that's good to know! Makes sense for what realloc is actually doing under the hood, but for whatever reason I had it in my head that it went both ways.